index.html

```diff
@@ -55,12 +55,65 @@
     </select>
   </div>
   </main>
-  <footer class="page-footer footer-light transparent">
+  <footer class="page-footer">
     <div class="row center for-mobile"> <a class="github-button" href="{{ site.githuburl }}" data-size="large">Fork</a> </div>
-    <div class="row">
-      <div class="col m6 offset-m3 center"> <img class="footer-logo" src="https://api.coala.io/en/latest/_static/images/coala_logo.svg" alt="coala"> <span class="footer-title">{{ site.organization }}</span> </div>
+    <div class="container">
+        <div class="row">
+            <div class="col m3 s12">
+                <h5 class="white-text fine">Site Links</h5>
+                <ul>
+                    <li><a class="grey-text text-lighten-3" href="http://coala.io/docs">Documentation</a></li>
+                    <li><a class="grey-text text-lighten-3" href="http://coala.io/newcomer">Newcomer Docs</a></li>
+                    <li><a class="grey-text text-lighten-3" href="http://coala.io/coc">Code of Conduct</a></li>
+                    <li><a class="grey-text text-lighten-3" href="http://blog.coala.io/">Blog</a></li>
+                </ul>
+            </div>
+            <div class="col m9 s12 center">
+              <div class="center">
+                <h3 class="fine white-text">
+                  <span>
+                    <img class="footer-logo" src="https://api.coala.io/en/latest/_static/images/coala_logo.svg" alt="coala">
+                  </span>
+                  <span>{{ site.organization }}</span>
+                </h3>
+              </div>
+              <div class="center footer-icons">
+                <span class="icon">
+                  <a href="https://twitter.com/coala_io" target="_blank" class="btn center">
+                    <i class="fa fa-twitter" aria-hidden="true">
+                      Follow us on Twitter
+                    </i>
+                  </a>
+                </span>
+                <span class="icon">
+                  <a href="https://gitter.im/coala/coala/gsoc" target="_blank" class="btn center">
```

Comment:
Technically these projects are a part of GSoC and hence this might be the correct gitter channel for the discussion, but I was thinking if it would be better to redirect the user to coala's gitter homepage `https://gitter.im/coala/` and from there a user could select gsoc channel for discussion. The benefit of this flow would be that the user would have a list of all the other(more active) channels as well. This debatable though, and so the decision is upto you :)

Reply:
IMO your idea would look great if this is implemented on main [coala website](coala.io) but for projects website i think redirecting to `gsoc` channel is OK because developers interested in GSoC visit more on this website,

Reply:
I agree with this link being the gsoc room - the reality is this website is mostly used for GSoC , and more noise in the gsoc room is OK, whereas more gsoc noise in the coala room is distracting.